### ISSUE TITLE
Hover affordance

### ISSUE THREAD

**Issue by laurakalbag** [statelyai/feedback#209]
### Details
There should be affordance via hover effects that shows what is clickable (e.g. state nodes, edges)
https://discord.com/channels/795785288994652170/912333666456313907/914844589628727316
https://discord.com/channels/795785288994652170/912333666456313907/912414827522195468
### Category

Editor

---
Originally posted on: 2022-06-20T22:50:04.448Z
By: David Khourshid
Previous votes: 2
Previous status: under review
